### PR TITLE
embedded issuers

### PR DESCRIPTION
**Pull request by plusema86**

### CODE REVIEW

File: lib/iden3comm/data/mappers/proof_request_filters_mapper.dart [hunk 1]
```diff
@@ -223,84 +223,48 @@ class ProofRequestFiltersMapper
...
-          return FilterEntity(
-              operator: FilterOperator.inList,
-              name: 'credential.credentialSubject.$field',
-              value: falseValues);
-        }
+      parsedValue = value;
+    } else {
+      return null;
+    }
+
```
Comment (5eeman) [inline lib/iden3comm/data/mappers/proof_request_filters_mapper.dart:238]: Maybe just add something like
```
final value = parsedValue ? trueValues : falseValues;
```

**Review by 5eeman — CHANGES_REQUESTED** [review]
One comment and ready to merge.

File: lib/iden3comm/data/mappers/proof_request_filters_mapper.dart (hunk 1 above)
Comment (plusema86) [reply lib/iden3comm/data/mappers/proof_request_filters_mapper.dart:238]: ok let me check if I can simplify this

Comment (plusema86) [reply lib/iden3comm/data/mappers/proof_request_filters_mapper.dart:250]: refactored

**Pull request merged**